nav2_docking/opennav_docking/src/docking_server.cpp

```diff
@@ -326,6 +329,7 @@ void DockingServer::dockRobot()
             }
             result->success = true;
             result->num_retries = num_retries_;
+            dock->plugin->stopDetectionProcess();
```

Comment:
Move after publishZeroVelocity() so the order is the same in all terminal conditions